CONTRIBUTING.md

```diff
@@ -20,6 +20,35 @@ should pick up any unit test and run it). There are also [scripts](scripts) for
 [linting](scripts/find-lint.sh) and doing a [build/test/lint
 run](scripts/build-test-lint.sh).
 
+## Continuous Integration
+
+When a Pull Request is submitted, continuous integration jobs are run
+automatically to ensure the code builds and is relatively well-written. Checks
+are run on [Buildkite](https://buildkite.com/matrix-dot-org/dendrite/) and
+[CircleCI](https://circleci.com/gh/matrix-org/dendrite/).
+
+If a job fails, click the "details" button and you should be taken to the job's
```

Comment:
May need to clarify the web page the button is on.

Reply:
Added an image. Does that help?

Worth noting that the checks will be expanded automatically on a failing PR.

Reply:
Yeah, that'll be *really* detailed.